.github/workflows/build.yml

```diff
@@ -17,10 +17,12 @@ jobs:
     - name: FE unit tests
       run: |
         cd frontend
+        npm install
```

Comment:
you might want to take a look at this https://docs.npmjs.com/cli/ci.html

Reply:
thanks, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/cd851298d2d5058b4f9b94c5e69b4e63ea390d0e

Reply:
and here :) https://github.com/salsita/foosball-rating/pull/154/commits/dfec6545ed6d9a98ad73fbd0ee640deb8ae7d52a